model/histogram/float_histogram.go

```diff
@@ -882,17 +911,6 @@ mergeLoop: // Merge together all buckets from the original schema that fall into
 	return true
 }
 
-// targetIdx returns the bucket index within i.targetSchema for the given bucket
-// index within i.schema.
-func (i *floatBucketIterator) targetIdx(idx int32) int32 {
-	if i.schema == i.targetSchema {
-		// Fast path for the common case. The below would yield the same
-		// result, just with more effort.
-		return idx
-	}
```

Comment:
This common case is now implemented at the higher level (`floatBucketIterator.Next`).